users-service/features/support/env.ls

```diff
@@ -8,7 +8,7 @@ require! {
 db = null
 get-db = (done) ->
   return done db if db
-  MongoClient.connect "mongodb://localhost:27017/space-tweet-users-test", N (mongo-db) ->
+  MongoClient.connect "mongodb://localhost:27017/space-tweet-db-test", N (mongo-db) ->
```

Comment:
Why do you change the name here? I think it should still be called "users", since it's the database for the users service.